go/database/log/queries/queries.sql

```diff
@@ -0,0 +1,31 @@
+-- name: GetCollectionForUpdate :one
+SELECT *
+FROM collection
+WHERE id = $1
+FOR UPDATE;
+
+-- name: InsertRecord :copyfrom
+INSERT INTO record_log (collection_id, "offset", record) values($1, $2, $3);
```

Comment:
Do we want to do batch insert?

Reply:
OK. Tha batch insert is handled in log_repository.

Reply:
yes it does batch insert, :copyFrom https://docs.sqlc.dev/en/stable/howto/insert.html#using-copyfrom